isis/src/docsys/build/DevExplore.xsl

```diff
@@ -0,0 +1,333 @@
+        <xsl:text>&#xa;</xsl:text>
+        </script>
+      </head>
+
+      <body>
+
+        <div class="isisMenu">
+         <xsl:call-template  name="writeMenu"/>
+        </div>
+
+        <div class="isisContent">
+
+        <a href="http://www.usgs.gov">
+        <img src="../assets/icons/littleVIS.gif" width="80" height="22" border="0" alt="USGS"/></a><br/>
+
+
+        <p style="margin-top:10px; padding-bottom:0px;">
+        ISIS Documentation</p>
+
+        <hr/>
+
+
+        <table width="100%" cellpadding="0" border="0" cellspacing="0">
+          <tr valign="top">
+            <td align="left">
+              <h1>
+                Contributor Explore in Detail 
+              </h1>
+              <p>
+                Creating and maintaining quality tools is not simple. ISIS contributors have
+                worked hard to create accurate references and guides to support future
+                contributions. If you encounter a specific case that could help future
+                contributors, feel free to submit documentation for this page.
+              </p>
+              <p>
+                On this page you will find detailed references and specialized guides for contributing to ISIS.
+              </p>
+              <p>
+              </p>
+            </td>
+            <td align="right" class="caption">
+            <a href="../index.html">Home</a>
+            </td>
+          </tr>
+        </table>
+
+
+<!-- API Reference​ -->
+    <hr/>
+    <h2>ISIS API Reference</h2>
+      <p>
+        The ISIS API reference is particularly useful if you are working on an application
+        or interested in using the ISIS library as a dependency.
+      </p>
+      <table class="tableTOC">
+
+        <!-- hardcoded links -->
+        <tr valign="top">
+          <th style="width:25%">
+            <a href="https://astrodiscuss.usgs.gov/">
+            astrodiscuss</a>
+          </th>
+          <td>
+          Ask or answer questions on out community discussion board.
+          </td>
+        </tr>
+        <tr valign="top">
+          <th style="width:25%">
+            <a href="https://github.com/USGS-Astrogeology/ISIS3/issues">
+            GitHub Issues</a>
+          </th>
+          <td>
+          Comment on bug reports or feature requests from other users.
+          </td>
+        </tr>
+        <tr valign="top">
+          <th style="width:25%">
+            <a href="https://github.com/USGS-Astrogeology/ISIS3/discussions?discussions_q=label%3ARFC">
+            Requests for Comment</a>
+          </th>
+          <td>
+          Provide feedback on major changes proposed for ISIS.
+          </td>
+        </tr>
+        <tr valign="top">
+          <th style="width:25%">
+            <a href="https://github.com/USGS-Astrogeology/ISIS_TC">
+            ISIS Technical Committee</a>
+          </th>
+          <td>
+          Get involved in project governance.
+          </td>
+        </tr>
```

Comment:
These links and descriptions are repeats from the DevStart landing page. Is this just a place holder?

Reply:
It is the same with the following two sections on this page

Reply:
All of the table stuff in this PR is just filler. I just want to get the pages up for right now. I'm either going to push the completed pages to this PR or put in a second PR with the content. I want to make sure everything is linking eachother correctly before spending more time on content.